libcsl_godot/src/ledger/transaction.rs

```diff
@@ -190,12 +259,96 @@ impl Utxo {
             ),
         )
     }
+
+    pub fn add_to_inputs_builder(&self, inputs_builder: &mut TxInputsBuilder) {
```

Comment:
I find it a bit odd that this functionality is a method of the `Utxo` struct, is there any reason for this to be here instead of in the `GTxBuilder`?